sopel/modules/etymology.py

```diff
@@ -40,8 +40,11 @@ def etymology(word):
     # @@ <nsh> sbp, would it be possible to have a flag for .ety to get 2nd/etc
     # entries? - http://swhack.com/logs/2006-07-19#T15-05-29
 
+    if len(word) == 0:
+        return 'No word added.'
+
     if len(word) > 25:
-        raise ValueError("Word too long: %s[…]" % word[:10])
+        return "Word too long: %s[…]" % word[:10]
```

Comment:
I would rather keep the `ValueError` logic here. Beside, the "no word" condition above could do exactly the same.

Reply:
```suggestion
        raise ValueError('Word too long: %s[…]' % word[:10])
```